src/sentry/models/event.py

```diff
@@ -42,6 +43,9 @@ class Event(Model):
 
     objects = BaseManager()
 
+    # Key to use when ordering a list of events manually
+    ordering_key = attrgetter('datetime', 'id')
```

Comment:
Yeah, the only other thing would be to randomly put it in the module? Or in `constants`? But that seems just as obscure too.